lib/gems/pending/appliance_console/cli.rb

```diff
@@ -207,6 +218,27 @@ def set_external_db
       config.post_activation
     end
 
+    DATE_REGEXP   = /^(2[0-9]{3})-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])/
+    TIME_REGEXP   = /^(0?[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])/
+
+    def set_date_time
+      date, time = options[:datetime].split("T")
```

Comment:
This is a weird interface. Maybe a better choice would be to have separate `--date` `--time` and `--timesync` options.

Plus this feels like it makes it impossible to set the date without also setting the time and vice versa.